python/cuml/test/dask/test_kneighbors_regressor.py

```diff
@@ -80,25 +80,22 @@ def dataset(request):
 def exact_match(output1, output2):
     l1, i1, d1 = output1
     l2, i2, d2 = output2
-    l2 = l2.squeeze()
+    i1, i2 = i1.squeeze(), i2.squeeze()
```

Comment:
Same as above. We should just let the `NearestNeighbors` tests catch discrepancies in the distances/indices and have the regressor/classifier just test the predictions.